examples/contrib/tensorflow/tensorflow_layer_matrix.py

```diff
@@ -0,0 +1,38 @@
+"""Example of how to convert an ODL operator to a tensorflow layer."""
+
+import tensorflow as tf
```

Comment:
I'm quite sure that is a nonproblem. With that said I'll change this whole thing to a submodule and split it, that allows better long-term extensions.